### ISSUE TITLE
--print produces empty output

### ISSUE THREAD

**Issue by slhck** [webpro/reveal-md#117]
Running with phantomjs 2.1.1 on OS X and Linux, both exhibit the same problem.

```
$ cat test.md
# hello

# world

$ reveal-md test.md --print
Attempting to print "test.md" to filename "test.pdf" as PDF.
Make sure to have PhantomJS installed (and in your path).
Export PDF: Reading reveal.js config [1/4]
Export PDF: Preparing print layout [2/4]
Export PDF: Preparing pdf [3/4]
Export PDF: Writing file [4/4]
Export PDF: Finished successfully!
```

The output only contains the background color, nothing else:

![image](https://user-images.githubusercontent.com/582444/27045314-c6e21c3e-4fa0-11e7-8e0e-a6bd3c21d558.png)

**Comment by webpro** [comment]
This seems to be related to the theme, I found that using a different theme renders a proper PDF:

```
reveal-md test.md --print --theme beige
```

I guess not all themes are compatible with the print styles. But it surprises me that the default theme now fails to render properly, as that is what's been tested all along. Also see #86 and #97 for related issues.

If anyone can shed a light on this, that would be very helpful.

**Comment by slhck** [comment]
Hm, even with `--theme beige` I get only one empty slide. And it's printed quite fast, meaning that it's not just styles, but apparently it doesn't even load or write any of the content.

I'm using PhantomJS 2.1.1 from Homebrew here.

I was able to successfully use [decktape](https://github.com/astefanutti/decktape) with a forked off version of PhantomJS to produce PDFs, which takes significantly longer for a few dozen slides.

**Comment by webpro** [comment]
Does it help to update Node.js to v6.11.0 and re-install reveal-md? I've had some issues with Node v4 and early Node v6.x.

**Comment by slhck** [comment]
Umm, I'm on Node 8.0.0, actually.

**Comment by webpro** [comment]
Not sure what exactly is going on, but I've just updated some dependencies for [v0.3.3](https://github.com/webpro/reveal-md/releases/tag/0.3.3) and it seems to render slides in PDF for Node v8. Please let me know if it works for you.

**Comment by slhck** [comment]
It worked, yes, thank you very much!

**Comment by ruippeixotog** [comment]
I'm having the same problem with [this presentation](https://github.com/ruippeixotog/intro-to-category-theory). I have all tools and dependencies in their latest versions:

```
❯ node -v
v8.1.2
❯ reveal-md --version
0.3.3
❯ phantomjs --version
2.1.1
```

Can you check if it happens with you too?

**Comment by webpro** [comment]
Yes, I'm also only one empty slide in the PDF. I'll reopen, but still not sure what the actual cause is.

**Comment by slhck** [comment]
I just tried to recompile the slides again and I also end up with an empty PDF, although it *definitely* used to work. The only thing I changed is that Homebrew upgraded me to Node 8.1.2 (from 8.0.0) in the meantime.

**Comment by breschke** [comment]
I am also having this problem, and it seems to work on and off again. Any new insight since the issue was reopened?

```
> node -v
v8.4.0
> reveal-md --version
0.4.1
> phantomjs --version
2.1.1
```

**Comment by webpro** [comment]
Sorry, but I'm actually never printing slides myself, so I'm not very interested in this. Happy to accept PRs!

**Comment by webpro** [comment]
This seems to have been fixed in #144, so I'm just going to close this issue. Feel free to report any issues.
